CONTRIBUTING.md

```diff
@@ -41,3 +41,39 @@ If you do not see an exact emoji match, use the best matching emoji.
     Updates Contributing.md with new template
 
     Resolves: #1234
+
+
+## Running Tests locally
+
+You can run all the tests inside each of the packages from the project root.
```

Comment:
Let's add a bit more context here: you can run all the tests locally OR you can go into an individual folder an run just that packages tests.